ynr/apps/people/migrations/0039_auto_20220428_1635.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 3.2.12 on 2022-04-28 15:35
+
+import django.contrib.postgres.indexes
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("people", "0038_remove_personimage_is_primary")]
+
+    operations = [
+        migrations.RemoveIndex(
+            model_name="person", name="name_vector_search_index"
+        ),
+        migrations.AddIndex(
+            model_name="person",
+            index=django.contrib.postgres.indexes.GinIndex(
```

Comment:
How did this change come about? Wondering if this is a missing migration from other work?

Reply:
@symroe Is this ok to commit with this PR, assuming it looks ok? I think it was missing from previous search-related work?